specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -3956,6 +4152,329 @@
       },
       "description": "Response from a List Indexes request. If successful, it includes the full definitions of all indexes."
     },
+	"Skillset": {
+		"properties": {
+			"name": {
+				"externalDocs": {
+					"url": "https://docs.microsoft.com/en-us/azure/search/cognitive-search-tutorial-blob"
+				},
+				"type": "string",
+				"description": "The name of the skillset."
+			},
+			"description": {
+				"type": "string",
+				"description": "The description of the skillset."
+			},
+			"skills": {
+				"type": "array",
+				"items": {
+					"$ref": "#/definitions/Skill"
+				},
+				"description": "A list of skills in the skillset."
+			},
+			"@odata.etag": {
+				"x-ms-client-name": "ETag",
+				"type": "string",
+				"description": "The ETag of the skillset."
+			}
+		},
+		"required": ["name",
+		"description",
+		"skills"],
+		"description": "A list of Cognitive Search skills."
+	},
+	"Skill": {
+		"discriminator": "@odata.type",
+		"properties": {
+			"@odata.type": {
+				"type": "string"
+			},
+			"description": {
+				"type": "string",
+				"description": "The description of the skill."
+			},
+			"context": {
+				"type": "string",
+				"description": "The context of the skill."
+			},
+			"inputs": {
+				"type": "array",
+				"items": {
+					"$ref": "#/definitions/InputFieldMappingEntry"
+				},
+				"description": "The inputs of the skill."
+			},
+			"outputs": {
+				"type": "array",
+				"items": {
+					"$ref": "#/definitions/OutputFieldMappingEntry"
+				},
+				"description": "The outputs of the skill."
+			}
+		},
+		"required": ["@odata.type",
+		"description",
+		"context",
+		"inputs",
+		"outputs"],
+		"description": "Abstract base class for skills."
+	},
+	"InputFieldMappingEntry": {
+		"properties": {
+			"name": {
+				"type": "string",
+				"description": "The name of the input."
+			},
+			"source": {
+				"type": "string",
+				"description": "The source of the input."
```

Comment:
This would be a useful place to have a link to documentation explaining the path syntax used in Cognitive Search.

Reply:
It seems that we don't have a documentation for the path syntax used in cognitive search. I have created a PBI 258046 for it. Also, since we have the documentation for each skill, I put links for the skills. (Synced up with Bruce on this through IM.)